File: leetcode/1672.py
"""
Runtime: 44 ms, faster than 98.19% of Python3 online submissions for Richest Customer Wealth.
Memory Usage: 14.4 MB, less than 29.91% of Python3 online submissions for Richest Customer Wealth.
"""


class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        return max([sum(r) for r in accounts])

File: leetcode/1672.cpp
// Runtime: 4 ms, faster than 84.18% of C++ online submissions for Richest Customer Wealth.
// Memory Usage: 8.2 MB, less than 9.24% of C++ online submissions for Richest Customer Wealth.
class Solution {
 public:
  int maximumWealth(vector<vector<int>> &accounts) {
    int res = INT_MIN;
    for (auto account : accounts) {
      res = std::max(res, std::accumulate(account.begin(), account.end(), 0));
    }

    return res;
  }
};
